RNReanimated.podspec

```diff
@@ -30,7 +30,7 @@ Pod::Spec.new do |s|
   s.license      = "MIT"
   # s.license    = { :type => "MIT", :file => "FILE_LICENSE" }
   s.author       = { "author" => "author@domain.cn" }
-  s.platforms    = { :ios => "9.0", :tvos => "9.0", :osx => "10.14" }
+  s.platforms    = { :ios => "10.0", :tvos => "9.0", :osx => "10.14" }
```

Comment:
Can we set `:ios` based on `react_native_minor_version`? This way we can preserve compatibility with iOS 9.0 on older RN versions.

Reply:
sure thing 🫡 done in https://github.com/software-mansion/react-native-reanimated/pull/4913/commits/bb531b3db12f1d2b7abc2d5b12dafef836bfd696